app/controllers/posts_controller.rb

```diff
@@ -16,10 +16,23 @@ def new
 
   def create
     @post = Post.new(post_params)
-    @post.author_id = Current.user.id
+    @post.author_id = current_user.id
```

Comment:
- [ ] nice work. please kindly ensure that the authorization Uses CanCanCan for Post authorization in Post Controller
eg
```ruby
class PostsController < ApplicationController
    load_and_authorize_resource

    ...
  end
```